calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
+  const numbers = [];
+
+  return {
+    add_number: function (number) {
+      numbers.push(number);
+    },
+    calculate: function () {
+      // Return zero if there are no numbers to calculate
+      if (numbers.length === 0) {
+        throw 'Cannot calculate a result because no numbers were given';
+      }
+
+      let answer = numbers[0];
+
+      let i = 0;
+      numbers.forEach(function (number) {
```

Comment:
Really though we don't care about 'i' all that much. All this is allowing the code to skip the first number (since it's already held in 'answer'). Perhaps slicing the first number off of the list _as part of_ the assignment to answer would be best.

Reply:
Yeah that's a better bit of advice. Maybe a while loop with a condition of `numbers.length > 0`  and use `shift` to remove elements